hapi-utils/src/test/java/com/hederahashgraph/builder/RequestBuilderTest.java

```diff
@@ -79,4 +145,616 @@ void testGetFileDeleteBuilder() throws InvalidProtocolBufferException {
+		assertEquals(transactionFee, tb.getTransactionFee());
+		assertEquals(nodeId.getAccountNum(), tb.getNodeAccountID().getAccountNum());
+		assertEquals(nodeId.getRealmNum(), tb.getNodeAccountID().getRealmNum());
+		assertEquals(nodeId.getRealmNum(), tb.getNodeAccountID().getShardNum());
+		assertEquals(accountId.getAccountNum(), tb.getTransactionID().getAccountID().getAccountNum());
+		assertEquals(accountId.getRealmNum(), tb.getTransactionID().getAccountID().getRealmNum());
+		assertEquals(accountId.getShardNum(), tb.getTransactionID().getAccountID().getShardNum());
+	}
+
+	@Test
+	void assertGetTxBodyBuilder() {
+		var transactionBody = RequestBuilder.getTxBodyBuilder(transactionFee, startTime,
+				transactionDuration, generateRecord, memo, accountId,
+				nodeId);
+
+		assertEquals(memo, transactionBody.getMemo());
+		assertEquals(transactionFee, transactionBody.getTransactionFee());
+		assertEquals(startTime, transactionBody.getTransactionID().getTransactionValidStart());
+		assertEquals(accountId, transactionBody.getTransactionID().getAccountID());
+		assertEquals(generateRecord, transactionBody.getGenerateRecord());
+		assertEquals(transactionDuration, transactionBody.getTransactionValidDuration());
+		assertEquals(nodeId, transactionBody.getNodeAccountID());
+	}
+
+	@Test
+	void assertGetTransactionBody() throws InvalidProtocolBufferException {
+		var accId = AccountID.newBuilder()
+				.setShardNum(0)
+				.setRealmNum(0)
+				.setAccountNum(1005)
+				.build();
+
+		var transaction = RequestBuilder.getAccountUpdateRequest(
+				accId,
+				accountId.getAccountNum(),
+				accountId.getRealmNum(),
+				accountId.getShardNum(),
+				nodeId.getAccountNum(),
+				nodeId.getRealmNum(),
+				nodeId.getShardNum(),
+				transactionFee, startTime, transactionDuration, generateRecord, memo, autoRenew);
+
+		var tb = TransactionBody.parseFrom(transaction.getBodyBytes());
+		assertEquals(memo, tb.getMemo());
+		assertEquals(generateRecord, tb.getGenerateRecord());
+		assertEquals(transactionFee, tb.getTransactionFee());
+		assertEquals(startTime, tb.getTransactionID().getTransactionValidStart());
+		assertEquals(transactionDuration, tb.getTransactionValidDuration());
+		assertEquals(transactionFee, tb.getTransactionFee());
+		assertEquals(nodeId.getAccountNum(), tb.getNodeAccountID().getAccountNum());
+		assertEquals(nodeId.getRealmNum(), tb.getNodeAccountID().getRealmNum());
+		assertEquals(nodeId.getShardNum(), tb.getNodeAccountID().getShardNum());
+		assertEquals(accountId.getAccountNum(), tb.getTransactionID().getAccountID().getAccountNum());
+		assertEquals(accountId.getRealmNum(), tb.getTransactionID().getAccountID().getRealmNum());
+		assertEquals(accountId.getShardNum(), tb.getTransactionID().getAccountID().getShardNum());
+	}
+
+	@Test
+	void assertGetFileCreateBuilder() throws InvalidProtocolBufferException {
+		Timestamp fileExpiration = Timestamp.newBuilder().setSeconds(123_456L).build();
+		List<Key> keyList = List.of();
```

Comment:
I think it is a good idea to not leave the keyList empty for the sake of the unit test passing.